docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -528,6 +528,42 @@ file suffix.
 
     :return bytes: Serialized PKCS12.
 
+PKCS7
+~~~~~
+
+.. currentmodule:: cryptography.hazmat.primitives.serialization.pkcs7
+
+PKCS7 is a format described in :rfc:`2315`, among other specifications. It can
+contain certificates, CRLs, and much more. PKCS7 files commonly have a ``p7b``,
+``p7m``, or ``p7s`` file suffix but other suffixes are also seen in the wild.
+
+.. note::
+
+    ``cryptography`` only supports parsing certificates from PKCS7 files at
+    this time.
+
+.. function:: load_pem_pkcs7_certificates(data)
+
+    .. versionadded:: 3.1
+
+    Deserialize a PEM encoded PKCS7 blob.
```

Comment:
This should describe the behavior in more detail. Specifically it should say that it will only return the certificates, even if the PKCS7 thingy contains other types of data.